sites/svelte.dev/src/routes/tutorial/[slug]/TableOfContents.svelte

```diff
@@ -33,7 +35,13 @@
 			{selected.chapter.title}
 		</span>
 
-		<select value={slug} on:change={navigate}>
+		<select
+			aria-label="{selected.section.title} / {selected.chapter.title}"
```

Comment:
The aria-label here should be labelling the control instead of what is currently selected (that's exposed separately). Maybe something like "Tutorial Chapter" instead.